definitions/service_plan_list.yml

```diff
@@ -0,0 +1,25 @@
+# yaml-language-server: $schema=.schema.yml
+beta: true
+location: internal/plugin/service/planlist
+datasource:
+  description: >
+    A list of service plans for a given project and service type.
+idAttribute:
+  fields:
+    - project
+    - service_type
+remove:
+  - service_plans/node_count
+  - service_plans/shard_count
+  - service_plans/max_memory_percent
+  - service_plans/backup_config
+  - service_plans/service_type
+operations:
```

Comment:
I have double-checked the issue https://github.com/aiven/terraform-provider-aiven/issues/2194
And turns out, most of these fields are on the feature request. So we shouldn't remove them.